### PR TITLE
Change sensor enum from ALL_CAPS to Pascal style

### PR DESCRIPTION
**Pull request by vauduong**
## Motivation and Context

Changing style for enumeration

## How Has This Been Tested

Compiled and built

## Types of changes

- [x ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/esp/sensor/Sensor.h [hunk 1]
```diff
@@ -19,22 +19,22 @@ namespace sensor {
...
-  PATH = 5,
-  GOAL = 6,
-  FORCE = 7,
-  TENSOR = 8,
-  TEXT = 9,
+  None = 0,
+  Color = 1,
+  Depth = 2,
+  Normal = 3,
+  Semantic = 4,
```
Comment (bigbike) [inline src/esp/sensor/Sensor.h:26]: I believe a couple of places use the `Semantic` and the `Depth` as well. But I did not see the changes in the files below.

**Comment by erikwijmans** [comment]
Make sure that your submodules match the ones on master.  Looks like they got reset.

**Comment by bigbike** [comment]
@erikwijmans : I believe she pulled the master first (git pull master), and merged the changes directly to her own branch WITHOUT compiling the new submodule commits first.

Do you know how she could drop the modifications to the submodules but keep the other useful changes? 
if we use `git reset`, then changes in the last commits are lost as well.

**Comment by vauduong** [comment]
Thanks, I think that worked to reset the submodules!

**Comment by vauduong** [comment]
Fixed, had to unfollow project on personal org in CircleCI and recreate changes in new branch to get CI tests to trigger in facebookresearch org

**Comment by bigbike** [comment]
Nice! You figured out the root cause. This is great!

**Pull request closed without merging**